csm_web/frontend/src/components/App.tsx

```diff
@@ -63,7 +65,7 @@ function Header(): React.ReactElement {
    * That is, it should always be active unless we're in a location prefixed by /resources
    */
   const schedulerActive: NavLinkProps["isActive"] = (match, location): boolean => {
-    return !location.pathname.startsWith("/resources");
+    return !(location.pathname.startsWith("/resources") || location.pathname.startsWith("/policies"));
```

Comment:
Here, since we're adding more header items, it may be better just to generalize this, and have a constant array of prefixes that we want to exclude

Reply:
Resolved, changed schedulerActive to loop through links array for scalability.